.env.1p.skate

```diff
@@ -0,0 +1,22 @@
+# MBTA Skate Secret Environment Variables
```

Comment:
Nit: It's throwing me off a bit that all of the other `.envrc` files are called `.envrc...`, but this one is `.env.1p...`. Could we call this `.envrc.1p.skate` instead?

Reply:
this isn't a `.envrc` file though, this is specifically a `.env` file, which does not use shell functions (i.e., `export`) and is only key=value mappings [as I currently understand it].

Reply:
That is a fair point that I hadn't noticed, and I withdraw my request